collector/src/main/java/com/bitmovin/analytics/data/HttpBackend.java

```diff
@@ -4,42 +4,43 @@
 import android.net.Uri;
 import android.util.Log;
 
-import com.bitmovin.analytics.CollectorConfig;
+import com.bitmovin.analytics.BitmovinAnalyticsConfig;
 import com.bitmovin.analytics.utils.DataSerializer;
 import com.bitmovin.analytics.utils.HttpClient;
+import okhttp3.Callback;
 
 public class HttpBackend implements Backend {
     private static final String TAG = "BitmovinBackend";
     private HttpClient httpClient;
     private String analyticsBackendUrl;
     private String adsAnalyticsBackendUrl;
 
-    public HttpBackend(CollectorConfig config, Context context) {
```

Comment:
I think you can stay with the `CollectorConfig` here as that's the configuration that configures how the collector works